test/TestUtilities/Test.Utility/TestDotnetCLiUtility.cs

```diff
@@ -233,6 +236,21 @@ private static void CopyRestoreArtifacts(string artifactsDirectory, string pathT
                 sourceFileName: typeof(Newtonsoft.Json.JsonSerializer).Assembly.Location,
                 destFileName: Path.Combine(pathToSdkInCli, "Newtonsoft.Json.dll"),
                 overwrite: true);
+
+            File.Copy(
+                sourceFileName: typeof(FileInfoBase).Assembly.Location,
+                destFileName: Path.Combine(pathToSdkInCli, Path.GetFileName(typeof(FileInfoBase).Assembly.Location)),
+                overwrite: true);
+
+            File.Copy(
+                sourceFileName: typeof(IFileInfo).Assembly.Location,
+                destFileName: Path.Combine(pathToSdkInCli, Path.GetFileName(typeof(IFileInfo).Assembly.Location)),
+                overwrite: true);
+
+            File.Copy(
+                sourceFileName: typeof(IChangeToken).Assembly.Location,
+                destFileName: Path.Combine(pathToSdkInCli, Path.GetFileName(typeof(IChangeToken).Assembly.Location)),
+                overwrite: true);
```

Comment:
Please put comments saying what files these are for. Or maybe remove the `using` statements at the top of the file, so we can see the namespace in the `typeof`.

FWIW, in the .NET 6 SDK, I already see the FilesystemGlobbing.dll, so if https://github.com/NuGet/Client.Engineering/issues/1271 gets completed, these file copies will probably not be needed.